modules/reserved-ips/variables.tf

```diff
@@ -0,0 +1,126 @@
+##############################################################################
+# VPC Variables
+##############################################################################
+
+variable "prefix" {
+  description = "The prefix that you would like to append to your resources. Value is only used if no value is passed for the `vpe_name` option in the `cloud_services` input variable."
+  type        = string
+  default     = "vpe"
+}
+
+variable "vpc_name" {
+  description = "Name of the VPC where the Endpoint Gateways will be created. This value is used to dynamically generate VPE names. Value is only used if no value is passed for the `vpe_name` option in the `cloud_services` input variable."
+  type        = string
+  default     = "vpc"
+}
+
+##############################################################################
+# SUBNET Variables
+##############################################################################
+
+variable "subnet_zone_list" {
+  description = "List of subnets in the VPC where gateways and reserved IPs will be provisioned. This value is intended to use the `subnet_zone_list` output from the Landing Zone VPC Subnet Module (https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc) or from templates using that module for subnet creation."
+  type = list(
+    object({
+      name = string
+      id   = string
+      zone = optional(string)
+      cidr = optional(string)
+    })
+  )
+  default = []
+}
+
+
+##############################################################################
+# VPE Variables
+##############################################################################
+
+variable "region" {
+  description = "The region where VPC and services are deployed"
+  type        = string
+  default     = "us-south"
+}
+
+variable "cloud_services" {
```

Comment:
It's only used for the naming of the reserved ips, so I would make this field more generic to take a list of strings. Also the description is not accurate.

Reply:
`vpe_name` maybe needed in [naming convention](https://github.com/terraform-ibm-modules/terraform-ibm-vpe-gateway/pull/415/files/b2eab2ad3856bf16d8445cb9ee3297f03efe3949#diff-9116576c0a8b977dbe986f7739366cfb48a4cfe0b1c3b7bf494929fd80965426R15) to match main module.